instrumentation/opentelemetry-instrumentation-logging/src/opentelemetry/instrumentation/logging/__init__.py

```diff
@@ -14,6 +14,23 @@
 
 # pylint: disable=empty-docstring,no-value-for-parameter,no-member,no-name-in-module
 
+"""
+The OpenTelemetry `logging` integration automatically injects tracing context into
+log statements.
+
+Usage
+-----
+.. code-block:: python
+
+    import logging
+
+    from opentelemetry.instrumentation.logging import LoggingInstrumentor
+
+    LoggingInstrumentor().instrument()
+
+    logging.warning('OTel test')
```

Comment:
It may be useful to add an example of the output after the code block

Reply:
Uhm I think we need an `export OTEL_PYTHON_LOG_CORRELATION=true` to have the instrumentation do something

Reply:
Thanks both for pointing that out. I had missed that. @emdneto I borrowed some sentences from the documentation to improve the examples. :)